src/styles/index.scss

```diff
@@ -1,3 +1,136 @@
+:root {
+  --link-color: #00acdc;
+  --color: #616070;
+  --bg-color: #fff;
+  --text-color: #060b35;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
```

Comment:
The width of the card is set to a fixed value of 200px. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use relative units like 'em' or 'rem' for widths to enhance the responsiveness of the design.